package/MDAnalysis/analysis/rms.py

```diff
@@ -711,6 +713,107 @@ def rmsd(self):
         return self.results.rmsd
 
 
+class SymmRMSD(AnalysisBase):
+    """
+    Compute symmetry-corrected RMSD for small molecules
+    """
+
+    def __init__(self, atomgroup, reference=None, select='all', ref_frame=0, **kwargs):
+        super().__init__(atomgroup.universe.trajectory, **kwargs)
+
+        self.atomgroup = atomgroup
+        self.reference = reference if reference is not None else self.atomgroup
+        self.ref_frame = ref_frame
+
+        select = process_selection(select)
```

Comment:
Can we move away from the pattern of passing in selection strings? IMHO more preferable is AtomGroups (which you can make with selection strings if that works for you)